lib/ui/src/modules/api/index.js

```diff
@@ -8,6 +8,7 @@ export default {
       name: 'STORYBOOK',
       url: 'https://github.com/storybooks/storybook',
       sortStoriesByKind: false,
+      resolveStoryHierarchy: storyName => [storyName],
```

Comment:
I guess we'll lose a serialized function after passing through the channel. 
We can consider options: 
- pass only JSON data here
- provide this API on the `manager` side (it'll be available in `addons.js`)